CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## v0.12.0
+
+### Device support
+- Added support for non-Snapdragon powered devices. Most chips on the market are supported now, 
+including Exynos by Samsung, Kirin by Huawei, Mediatek, etc.
+
+### Vision
+- Added `Japan` country
+- Added new `SignTypes` : 
+`InformationRestrictedParking`, `RegulatorySchoolZone`, `RegulatoryBicyclesAndPedestriansCrossing`,
+`RegulatoryNoBusesAndHeavyVehicles`
+- Added `setProgress`/`getProgress` to VisionReplayManager to control session playback progress
+- Added `getDuration` to VisionReplayManager
+- Changed VisionReplayManager's behaviour:
+    method start continues session replay from the current progress
```

Comment:
Do we have it documented anywhere else? It seems I see it for the first time.

Reply:
Yeah in `VisionReplayManager.start` we do have `If [VisionReplayManager] was stopped, then [start] will resume reading the session from the moment it was stopped.`